package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "revai-node-sdk",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
You haven't released the axios package fix yet? Why is this shown as a version increase?

Reply:
If you haven't released the axios fix, then you can release these two tickets together when we are ready to ship.